### ISSUE TITLE
Has the How WCAG and other Accessibility Guidelines Apply to mobile document been superseded by WCAG 2.1 or just in need of an update?

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#371]
What is the status of this document in context of WCAG 2.1 release?  It seems like some guidelines have changed but other guidelines not yet adopted in WCAG 2.x.  Do we need to update this document with the changes from 2.1?

https://www.w3.org/TR/mobile-accessibility-mapping/

**Comment by alastc** [comment]
What is the Mobile TF's view on the doc?

**Comment by alastc** [comment]
Could @KimPatch or @kwahlbin comment on the status of the "mobile accessibility mapping" now?

**Comment by detlevhfischer** [comment]
@KimPatch @kwahlbin @alastc The doc is clearly out of date though nothing seems outright wrong.

- 3.1 Keyboard Control for Touchscreen Devices should include 2.5.6 Concurrent Input Mechanisms
- 3.2 Touch Target Size and Spacing should include 2.5.5 Target Size
- 3.3 Touchscreen Gestures could / should be expanded to include 2.5.1 Pointer gestures and 2.5.2 Pointer Cancellation
- 3.4 Device Manipulation Gestures should be expanded to include 2.5.4 Motion Actuation
- 4.1 Changing Screen Orientation (Portrait/Landscape) should include 1.3.4 Orientation
- 4.2 Consistent Layout should reference 1.4.10 Reflow, especially the aspect of "without loss of information or functionality" 

The update, if desired, is not something that would happen overnight - but I guess within the Mobile a11y TF we can discuss 

1. whether we feel it makes sense to produce an update that retains the old format
2. and if we think it worthwhile, how to set up this task and who's going to contribute.

**Comment by alastc** [comment]
I think the options would be either:

- Update the doc in the light of 2.1
- Mark it as... not obscelete per-se, but perhaps dated and not going to be updated.
- Mark it as obscelete and take relevent guidance not covered by 2.1 for a new doc, e.g. supplimental guidance that could work with the COGA & LV docs.

**Comment by patrickhlauke** [comment]
My personal preference would be option three "Mark it as obsolete and take relevent guidance not covered by 2.1 for a new doc". There are still aspects/gaps that 2.1 only partially addresses, but it would be good to start with a clean slate. And ideally, some of the "for mobile, you need to interpret it this way" could flow into the actual understanding docs for relevant SCs in core WCAG 2.1 understanding, rather than being relegated to a separate document.

**Comment by DavidMacDonald** [comment]
@patrickhlauke  +1

**Comment by jake-abma** [comment]
@patrickhlauke +1

**Comment by detlevhfischer** [comment]
I am not sure of the value of a document covering "relevant guidance not covered by 2.1" - something like "to be read in conjunction with 2.1" - that could be confusing. The attraction of the old document was that tried to offer a comprehensive guide for mobile design without caring for the WCAG 2.1 structure (which we know has many overlaps and has grown somewhat more inconsistent now that new SCs have appeared, quasi bolted on, with a promise of a revamp in Silver). I am not sure whether it is worth the effort to work on something which looks like a make-shift document until the point when the grand new scheme, Silver, will be ready. There is still work to be done on Techniques for 2.1 and clarifying how the new SCs should be interpreted - I get questions from testers about them on a very regular basis. So this is where I feel our time is best spent, right now.

**Comment by alastc** [comment]
> WCAG 2.1 structure (which we know has many overlaps and has grown somewhat more inconsistent now that new SCs have appeared, quasi bolted on, with a promise of a revamp in Silver)

To be fair, I don't think there is a single structure that would be easy & understandable, the advantage Silver has is taking a more database-approach and having multiple structures in one. But I digress...

If everyone has plenty to do, and this is clearly a WCAG 2.**0** oriented document, perhaps we just leave it and focus on 2.1 techniques, 2.2 & silver?

**Comment by patrickhlauke** [comment]
WFM

if there are any major gaps that we see needing to be addressed (or more, ways in which some 2.1 SCs - including legacy 2.0 ones - need a bit more interpretation for some mobile browser/OS specific quirks), then it possibly makes more sense to look at expanding the relevant core understanding documents, rather than relegating to a separate "for mobile" document (this would also help to underline the fact that in most areas, the line between "desktop" and "mobile/tablet" is far more smudged than before)

**Comment by fstrr** [comment]
Closing as this was labelled as "Inactive - close soon" in 2019.
